src/.vuepress/styles/index.scss

```
@@ -0,0 +1,7 @@
+h2 {
```

Might wanna future-proof it? 😁 

```suggestion
h1, h2, h3, h4, h5 {
```

---

I'd honestly rather it be more specific than less. This way if the formatting doesn't match someone has to go make the conscious decision to make the change.